pkg/referrerstore/oras/cache_test.go

```diff
@@ -0,0 +1,178 @@
+	return ocispecs.ReferenceManifest{}, nil
+}
+
+func (m *mockBase) GetSubjectDescriptor(ctx context.Context, subjectReference common.Reference) (*ocispecs.SubjectDescriptor, error) {
+	return testDesc, nil
+}
+
+func TestCreateCachedStore(t *testing.T) {
+	if _, err := createCachedStore(base, conf); err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+}
+
+func TestName(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	name := store.Name()
+	if name != testName {
+		t.Fatalf("expect name: %s, got %s", testName, name)
+	}
+}
+
+func TestGetConfig(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	conf := store.GetConfig()
+	if !reflect.DeepEqual(conf, testStoreConfig) {
+		t.Fatalf("expect config: %+v, got %+v", testStoreConfig, conf)
+	}
+}
+
+func TestGetBlobContent(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	blob, err := store.GetBlobContent(context.Background(), testReference, testDigest)
+	if err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+	if !reflect.DeepEqual(blob, testBlob) {
+		t.Fatalf("expect blob: %v, got %v", testBlob, blob)
+	}
+}
+
+func TestGetSubjectDescriptor(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	desc, err := store.GetSubjectDescriptor(context.Background(), testReference)
+	if err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+	if !reflect.DeepEqual(testDesc, desc) {
+		t.Fatalf("expect desc: %v, got %v", desc, testDesc)
+	}
+}
+
+func TestListReferrers_CacheHit(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	result, _ := store.ListReferrers(context.Background(), testReference, []string{}, testNextToken1, nil)
+
+	time.Sleep(time.Duration(ttl-5) * time.Second)
+
+	cachedResult, err := store.ListReferrers(context.Background(), testReference, []string{}, testNextToken2, nil)
+	if err != nil {
+		t.Fatalf("err should be nil, but got %v", err)
+	}
+	if !reflect.DeepEqual(result, cachedResult) {
+		t.Fatalf("cached result: %+v is different from result: %+v", cachedResult, result)
+	}
+}
+
+func TestListReferrers_CacheMiss(t *testing.T) {
+	store, _ := createCachedStore(base, conf)
+
+	result, _ := store.ListReferrers(context.Background(), testReference, []string{}, testNextToken1, nil)
+
+	time.Sleep(time.Duration(ttl+5) * time.Second)
+
+	cachedResult, err := store.ListReferrers(context.Background(), testReference, []string{}, testNextToken2, nil)
+	if err != nil {
+		t.Fatalf("err should be nil, but got %v", err)
+	}
+	if reflect.DeepEqual(result, cachedResult) {
+		t.Fatalf("cached result: %+v should be different from result: %+v", cachedResult, result)
+	}
+}
+
+func TestToCacheConfig(t *testing.T) {
+	conf, err := toCacheConfig(pluginConfig)
+	if err != nil {
+		t.Fatalf("expect no error, got %v", err)
+	}
+	if !reflect.DeepEqual(conf, conf) {
+		t.Fatalf("expect %v, got %v", conf, conf)
+	}
```

Comment:
This test is incorrect. `conf` variable is reinitialized here and the var defined in var block will be replaced, so this is essentially checking the same value twice.

Change `conf` to something else in this test or change the variable name in the var block above

Reply:
Thanks for catching it! It was renamed in a previous commit and I forgot to update it.